CHANGELOG.md

```diff
@@ -1,172 +1,58 @@
-- BUG-000165913: Fixed an issue where the stationary property on the MapView does not change value when zooming using a double tap and drag gesture on a mobile device.
-- BUG-000166439: Fixed an issue where the icon thumbnail for `ListItemPanel.image` had no size.
-- BUG-000166831: Fixed an issue where the `MapView.hitTest()` does not return a hit for a `GraphicsLayer` when the graphics have `TextSymbol` with `lineWidth`.
-- BUG-000166346: Fixed an issue where polygon features with newlines in the label expression did not display correctly.
-- BUG-000167231: Fixed an issue where a CIMSymbol with CIMGeometricEffectDashes was having performance issues at higher scales.
-- [Esri Community - 1403039](https://community.esri.com/t5/arcgis-javascript-maps-sdk-questions/4-29-breaking-change-mapview-goto-w-rotation/m-p/1403039): Fixed an issue where MapView.goTo would zoom out when only the rotation was set in the target.
-- [Esri Community - 1407481](https://community.esri.com/t5/arcgis-javascript-maps-sdk-questions/first-calls-to-sendmessagetoclient-not-used/m-p/1407481): Fixed an issue where the first call to StreamLayer.sendMessageToSocket is not applied on initial load.
-- [Esri Community - 1413086](https://community.esri.com/t5/arcgis-javascript-maps-sdk-questions/4-29-memory-leak-on-streamlayer/m-p/1413086): Fixed an issue where the client-side `StreamLayer` was causing a memory leak.
-- Fixed an issue where setting popupEnabled to false was not properly honored for LayerSearchSource.
-- The Features widget is out of beta.
-- VideoElement is now supported on iPhones.
-
 ## Deprecations
 
-The following are deprecated and will be removed in a future release. For anything deprecated in 4.29 and earlier, additional information and links are in the [release notes](https://developers.arcgis.com/javascript/latest/release-notes/#deprecated-classes-properties-methods-events).
-
-- The AMD npm package [`arcgis-js-api`](https://www.npmjs.com/package/arcgis-js-api) is deprecated at version 4.29. This is related to dropping local build support for legacy Dojo 1 and RequireJS. Use [@arcgis/core](https://developers.arcgis.com/javascript/latest/es-modules/) ES modules or plan on moving to [components (beta)](https://developers.arcgis.com/javascript/latest/components/) instead.
-- The npm package [`@arcgis/cli`](https://www.npmjs.com/package/@arcgis/cli) is deprecated at version 4.29. Plan on moving to [components (beta)](https://developers.arcgis.com/javascript/latest/components/) and Calcite.
-- The [`esri-loader`](https://github.com/Esri/esri-loader/blob/master/README.md) library is deprecated at version 4.29. Use [@arcgis/core](https://developers.arcgis.com/javascript/latest/es-modules/) ES modules or plan on moving to [components (beta)](https://developers.arcgis.com/javascript/latest/components/) instead.
```

Comment:
Shouldn't these still be listed here?

Reply:
Thanks for the reminder. I needed to move them from Deprecations to the Breaking Changes section. Done.